### PR TITLE
Allow updating the banner design status through the UI

### PR DESCRIPTION
**Pull request by tjmw**
## What does this change?

Saves the banner design status to the server/Dynamo when toggled in the UI.

Screenshot 2023-09-19 at 09 15 23

Currently the status doesn't have any effect on anything, this will happen in a future PR.

## How to test

Update the status of a banner design, see the status change in Dynamo.

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: public/src/components/channelManagement/bannerDesigns/LiveSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```
Comment (tomrf1) [inline public/src/components/channelManagement/bannerDesigns/LiveSwitch.tsx:1]: would it be possible to reuse this one?
https://github.com/guardian/support-admin-console/blob/main/public/src/components/shared/liveSwitch.tsx

Comment (tjmw) [reply public/src/components/channelManagement/bannerDesigns/LiveSwitch.tsx:1]: Oh ha! I hadn't spotted that one. This code was adapted from the test live switch (which has modal behaviour I didn't want, hence not using it). But yeah that looks identical, thanks I'll update.

Comment (tjmw) [reply public/src/components/channelManagement/bannerDesigns/LiveSwitch.tsx]: 👍  Fixed in 4b9fba582b9c4bd3495cae97fd79d53277131f3d.

**Review by tomrf1 — APPROVED** [review]
great!

**Pull request merged**